### DISCUSSION TITLE
dto中入参注解的添加把方法也加上了注解

### DISCUSSION THREAD

**Discussion by tzxylao** [babyfish-ct/jimmer#857] [Q&A]
specification SpaceUserRootSpecification {
    @QueryParam("account")
    eq(account)
}

我定义了这个一个specification，我的本意是类入参属性account上有这个@QueryParam("account")注解，但是自动生成的类对应的方法上也加上了这个注解，我不希望它生成
<img width="606" alt="image" src="https://github.com/user-attachments/assets/61b9b4af-241d-49e5-90c0-2535378436a8" />
有什么办法吗？

**Comment by babyfish-ct** [comment]
目前没什么办法，很多第三库，有的需要修饰字段才能运行，有的需要修饰方法才能运行。无数个bug提交后，被逼无奈，毛了，干脆都生成
